spec/controllers/api/irs_attempts_api_controller_spec.rb

```diff
@@ -40,6 +41,14 @@
   let(:existing_event_jtis) { existing_events.map(&:first) }
 
   describe '#create' do
+    before do
+      Aws.config[:s3] = {
+        stub_responses: {
+          get_object: { body: Base64.strict_encode64("{}") }
+        },
+      }
+    end
```

Comment:
Can we add specs that check the behavior when the feature flag is on + off? or like other cases when missing the `IrsAttemptApiLogFile` db record?

Reply:
Done!  Please feel free to double-check!